internal/cli/opsmanager/admin/backup/s3/list.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var listTemplate = `ID	URI	SSL	LOAD FACTOR	S3 AUTH METHOD{{range .Results}}
```

Comment:
[non blocking] could be just AUTH METHOD given the scope of the command but I don't feel strong about it
```suggestion
var listTemplate = `ID	URI	SSL	LOAD FACTOR	AUTH METHOD{{range .Results}}
```

Reply:
I am going to change this in the next command